### PR TITLE
feat: reply to a post

### PR DESCRIPTION
**Pull request by iuricmp**
This PR allows users to reply a post.

### CODE REVIEW

**Comment by iuricmp** [comment]
Please, review this PR after merging https://github.com/gnolang/gnosocial/pull/48

File: mobile/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# - localhost:36657
+# - testnet.gno.berty.io:36657
+EXPO_PUBLIC_GNO_REMOTE=localhost:36657
```
Comment (jefft0) [inline mobile/.env:3]: Should the default be the Berty test node?

**Comment by jefft0** [comment]
Resolved by updating the gnosocial on the Berty test node.

**Review by jefft0 — APPROVED** [review]
Tested on Android and iOS simulators. Needed features for a future PR:
* Click on a "reply to a reply" to view the nested replies
* When the app shows the other user's posts when viewing their profile, support reply there.

**Pull request merged**